octobot_trading/exchanges/connectors/cryptofeed_websocket_connector.py

```diff
@@ -377,37 +387,60 @@ def _subscribe_candle_feed(self):
                                  candle_interval=time_frame.value,
                                  symbols=self.filtered_pairs,
                                  log_message_on_error=True,
-                                 channels=[cryptofeed_constants.CANDLES],
-                                 callbacks={cryptofeed_constants.CANDLES: self.candle_callback})
+                                 channels=self.CANDLE_CHANNELS,
+                                 callbacks=self.callbacks)
             self.logger.debug(
                 f"Subscribed to the {time_frame.value} time frame OHLCV for {', '.join(self.filtered_pairs)}")
 
-    def _subscribe_all_pairs_feed(self):
+    def _subscribe_pairs_feeds(self):
         """
         Subscribes all time frame unrelated feeds
         """
+        self._subscribe_traded_pairs_feed()
+        self._subscribe_watched_pairs_feed()
+
+    def _subscribe_traded_pairs_feed(self):
+        """
+        Subscribes all time frame unrelated feeds for traded pairs
+        """
         if self.channels:
             self.client.add_feed(self.get_feed_name(),
                                  symbols=self.filtered_pairs,
+                                 log_message_on_error=True,
                                  channels=self.channels,
                                  callbacks=self.callbacks)
             for channel in self.channels:
                 self.logger.debug(f"Subscribed to {channel}")
 
+    def _subscribe_watched_pairs_feed(self):
+        """
+        Subscribes all time frame unrelated feeds for watched pairs
+        """
+        if self.watched_pairs:
```

Comment:
I think we should also subscribe to `self.WATCHED_PAIR_CHANNELS` for `self.filtered_pairs` otherwise regular pairs will never subscribe on `self.WATCHED_PAIR_CHANNELS`.
Or add them to `self.watched_pairs` in `_add_pair` ?

Reply:
self.WATCHED_PAIR_CHANNELS is a part of self.channels

Reply:
So filtered_pairs subscribe on self.WATCHED_PAIR_CHANNELS and many more than watched_pairs

Reply:
Got it, great ! Should we filter `self.WATCHED_PAIR_CHANNELS` to remove unsupported ones though ?